src/woo_publications/conf/base.py

```diff
@@ -80,6 +81,25 @@
     "https://identifier.overheid.nl/tooi/def/thes/kern/c_816e508d"
 )
 
+# Image field validator settings
+ALLOWED_IMG_EXTENSIONS = config(
+    "ALLOWED_IMG_EXTENSIONS",
+    default=[
+        "jpg",
+        "jpeg",
+        "png",
+        "gif",
+        "webp",
+    ],
+)
+assert set(ALLOWED_IMG_EXTENSIONS) <= set(
+    validators.get_available_image_extensions()
+), "img file type not supported"
+
+MAX_FILE_SIZE = config("MAX_FILE_SIZE", default=1_000_000)
+MAX_FILE_HEIGHT = config("MAX_FILE_HEIGHT", default=300)
+MAX_FILE_WIDTH = config("MAX_FILE_WIDTH", default=300)
```

Comment:
Probably needs some extra context, but I don't know if we want to go this route or not.